docker/network.go

```diff
@@ -168,9 +168,8 @@ func (n *Network) delete(ctx context.Context, c *Component) error {
 }
 
 func createNetworkIfNotExist(cli *client.Client, name, driver string) (string, error) {
-	res, err := cli.NetworkCreate(context.Background(), name, types.NetworkCreate{
-		CheckDuplicate: true,
```

Comment:
Did we remove this line on purpose?

Reply:
yes the flag is deprecated and is true by default